app/pages/sections/order-items/catalogue-solutions/order-item/flat/ondemand/manifest.json

```diff
@@ -0,0 +1,146 @@
+{
+  "questions": {
+    "price": {
+      "id": "price",
+      "mainAdvice": "What price have you agreed? (£)",
+      "unit": "{{ unit | lower }}",
+      "expandableSection": {
+        "dataTestId": "view-section-price-id",
+        "title": "What price should I enter?",
+        "innerComponent": "Enter the price you've agreed with the supplier. We've included the list price, but this can be changed if required."
+      }
+    },
+    "quantity": {
+      "id": "quantity",
+      "mainAdvice": "How many do you think you'll need?",
+      "rows": 3,
+      "expandableSection": {
+        "dataTestId": "view-section-quantity-id",
+        "title": "What quantity should I enter?",
+        "innerComponent": "Enter the estimated quantity you think you'll need either per month or per year."
+      }
+    },
+    "selectEstimationPeriod": {
+      "id": "selectEstimationPeriod",
+      "type": "radio",
+      "mainAdvice": "Over what period did you estimate the quantity?",
+      "options": [
+        {
+          "value": "month",
+          "text": "Per month"
+        },
+        {
+          "value": "year",
+          "text": "Per year"
+        }
+      ],
+      "expandableSection": {
+        "dataTestId": "view-section-estimation-period-id",
+        "title": "What period should I enter?",
+        "innerComponent": "This should be based on how you estimated the quantity you want to order."
+      }
+    }
+  },
+  "addPriceTable": {
+    "columnInfo": [
+      {
+        "data": "What price have you agreed? (£)",
+        "width": "50%",
+        "classes": "nhsuk-u-font-size-16"
+      },
+      {
+        "data": "Unit of order",
+        "width": "50%",
+        "classes": "nhsuk-u-font-size-16"
+      }
+    ],
+    "cellInfo": {
+      "price": {
+        "question": {
+          "type": "input",
+          "id": "price"
+        },
+        "classes": "nhsuk-input--width-10",
+        "expandableSection": {
+          "dataTestId": "view-section-input-id",
+          "title": "What price should I enter?",
+          "innerComponent": "You can change the list price if you've agreed a different rate with the supplier."
+        }
+      },
+      "unitOfOrder": {
+        "dataTestId": "unit-of-order"
+      }
+    }
+  },
+  "solutionTable": {
+    "columnInfo": [
+      {
+        "data": "Recipient name (ODS code)",
+        "width": "33%",
+        "classes": " nhsuk-u-padding-bottom-5 nhsuk-u-font-size-20"
+      },
+      {
+        "data": "Quantity {{ estimationPeriod }}",
```

Comment:
Similar to above, I think we'll need to set the selected estimation period in the `contextCreator`. However, we won't be able to do that until we have the previous page in the flow (quantity and estimation period) that @nehajain008 is currently working on. Therefore, I think it will be fine to leave this as is for now as a prompt.